### PR TITLE
docs: update for CLI

### PR DESCRIPTION
**Pull request by AndersDJohnson**
Creates a top-level CLI landing page and section on the docs site.

Updates the CLI package README with similar to the mockyeah README, which shows the help output but mostly just links to the new CLI section of the docs site.

Adds `playAll` command to some places it was not added to docs with #179, and in the CLI index bin.

### CODE REVIEW

File: packages/mockyeah-cli/README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # mockyeah-cli
 
-Command Line Utility for [mockyeah](https://github.com/ryanricard/mockyeah)
+Command line utility for [mockyeah](https://github.com/ryanricard/mockyeah),
+a powerful service mocking, recording, and playback utility.
 
-Installation:
+<img src="docs/book/logo/mockyeah.png" height="200" />
```
Comment (ryanricard) [inline packages/mockyeah-cli/README.md]: This path doesn't appear to be correct.

Comment (AndersDJohnson) [reply packages/mockyeah-cli/README.md]: It should work once published, it’s the same as the mockyeah package's one.

Comment (ryanricard) [reply packages/mockyeah-cli/README.md]: 👍

**Comment by ryanricard** [comment]
Thanks for taking care of this, @AndersDJohnson.

**Pull request merged**